code/modules/mob/living/carbon/alien/humanoid/emote.dm

```diff
@@ -76,7 +76,7 @@
 			playsound(src.loc, 'sound/voice/hiss5.ogg', 40, 1, 1)
 
 		if (act == "hiss")
-			playsound(src.loc, 'sound/voice/hiss3.ogg', 40, 1, 1)
+			playsound(src.loc, "hiss", 40, 1, 1)

```

Comment:
I really don't know byond code, but this works fine. What's different about get_sfx()?